src/app/application-form/application-form.component.scss

```diff
@@ -65,3 +69,76 @@ mat-error {
     color: var(--main-white);
   }
 }
+.ng-star-inserted {
+  font-size: 16px;
+}
+
+.mat-mdc-form-field-error {
+  display: flex !important;
+  align-items: center !important;
+  padding-top: 4px;
+  line-height: 1 !important;
+}
+
+::ng-deep .mat-mdc-button .mat-mdc-button-ripple,
```

Comment:
For the future - we must remember, that `::ng-deep` is a deprecated selector and will be removed in a future version of Angular.
If we want to style Material elements - we will have to add global styles (like a class that changes Material element styles) in the future.